web/src/components/admin/ModerationSettings/ModerationSettings.tsx

```
@@ -0,0 +1,153 @@
+              }
+
+              function handleNewWordSubmit() {
+                const trimmed = newWord.trim();
+                if (trimmed === "") return;
+                const newList = uniq([...(field.value ?? []), trimmed]);
+                field.onChange(newList);
+                setNewWord("");
+              }
+
+              function handleRemoveWord() {
+                const newList = field.value?.filter((word) => word !== word);
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)